src/test/regress/expected/isolation_shard_rebalancer_progress.out

```diff
@@ -0,0 +1,123 @@
+Parsed test spec with 3 sessions
+
+starting permutation: s2-lock-1 s2-lock-2 s1-rebalance-c1 s3-progress s2-unlock-1 s3-progress s2-unlock-2 s3-progress s1-commit s3-progress
+master_set_node_property
+
+
+step s2-lock-1:
+ SELECT pg_advisory_lock(29279, 1);
+
+pg_advisory_lock
+
+
+step s2-lock-2:
+ SELECT pg_advisory_lock(29279, 2);
+
+pg_advisory_lock
+
+
+step s1-rebalance-c1:
+ BEGIN;
+ SELECT * FROM get_rebalance_table_shards_plan('colocated1');
+ SELECT rebalance_table_shards('colocated1');
+ <waiting ...>
+step s3-progress: 
+ set client_min_messages=NOTICE;
+ SELECT
+  table_name,
+  shardid,
+  shard_size as source_shard_size,
+  sourcename,
+  sourceport,
+  target_shard_size,
+  targetname,
+  targetport,
+  progress
+ FROM get_rebalance_progress();
+
+table_name     shardid        source_shard_sizesourcename     sourceport     target_shard_sizetargetname     targetport     progress
+
+colocated1     1500001        49152          localhost      57637          0              localhost      57638          1
+colocated2     1500005        376832         localhost      57637          0              localhost      57638          1
+colocated1     1500002        196608         localhost      57637          0              localhost      57638          0
+colocated2     1500006        8192           localhost      57637          0              localhost      57638          0
+step s2-unlock-1:
+ SELECT pg_advisory_unlock(29279, 1);
+
+pg_advisory_unlock
+
+t
+step s3-progress:
+ set client_min_messages=NOTICE;
+ SELECT
+  table_name,
+  shardid,
+  shard_size as source_shard_size,
+  sourcename,
+  sourceport,
+  target_shard_size,
+  targetname,
+  targetport,
+  progress
+ FROM get_rebalance_progress();
+
+table_name     shardid        source_shard_sizesourcename     sourceport     target_shard_sizetargetname     targetport     progress
+
+colocated1     1500001        0              localhost      57637          49152          localhost      57638          2
+colocated2     1500005        0              localhost      57637          376832         localhost      57638          2
```

Comment:
funky, sizes being 0, means they are already removed?

Is the control pannel happy with zero values in the source? Will it not skew the calculation of the progress.
How would that hold for a shard that moves twice? `a -> b -> c`

At some point the `a->b` move will represent the progress is completed (2?) but might show `source_shard_size` and `target_shard_size` of both `0` if the `b -> c` move also finished and removed the shard from node `b`.

Reply:
I'll take a look at some control plane source code to be sure. However, I don't think they cannot have a "percentage completed" before this PR. Since this PR adds the target_shard_size column.

I also considered if we might want to show the target size if the source size is 0 after a move. The main reason I didn't do this is: I thought it would be useful to see that a shard is not being removed on the source, because we're also planning to enable `citus.defer_drop_after_shard_move` by default.